src/Raven.Client/Documents/Session/IAttachmentsSessionOperationsAsync.cs

```diff
@@ -30,6 +31,11 @@ public interface IAttachmentsSessionOperationsAsync : IAttachmentsSessionOperati
         /// </summary>
         Task<AttachmentResult> GetAsync(object entity, string name, CancellationToken token = default);
 
+        /// <summary>
+        /// Returns Enumerator of KeyValuePairs of attachment name and stream.
+        /// </summary>
+        Task<IEnumerator<AttachmentEnumeratorResult>> GetAsync(IEnumerable<KeyValuePair<string, string>> attachments, CancellationToken token = default);
```

Comment:
create a dedicated type instead of using `KeyValuePair`. KeyValuePair is not extendable, it will be hard to add something in the future. Also it is hard to grasp what Key and what Value is.

The type should have ctor that fills all properties. Properties should have `private set`.

Reply:
done